src/main/java/bob/DeadlineCommand.java

```diff
@@ -0,0 +1,57 @@
+package bob;
```

Comment:
I noticed that all the code files are packaged under "bob".  Perhaps create separate directories/packages to group related code files together?